### DISCUSSION TITLE
Debugging missing changelogs for Rust projects

### DISCUSSION THREAD

**Discussion by lfrancke** [renovatebot/renovate#13720] [Request Help]
We're evaluating Renovate as a replacement for Dependabot.
We're a Rust shop and unfortunately most PRs are missing changelogs.

See the [Renovate version](https://github.com/stackabletech/druid-operator/pull/98) and the [Dependabot version](https://github.com/stackabletech/druid-operator/pull/117). I looked at the renovate source and found the `release-notes.ts` file which has ample logging, unfortunately at the `trace` level which is not exposed in the UI.

Any suggestions on how to debug this?
I'm not a node expert and have no idea how to get this app running locally and how to test this behavior.

**Comment by rarkins** [comment]
Actually I think missing release notes for Rust is a known limitation right now as we were encountering issues with rate limits from the crates registry. We should aim to add this back soon.

**Reply by lfrancke** [reply]
Thank you for the quick reply again.
Can you point me at some code/issues where this is disabled?
I'd be happy to try and get this enabled again (and potentially talk to the crates.io folks about rate limit increases as well)

**Reply by rarkins** [reply]
The issue to track is #3486. I'm adding a comment in there now. You can see the original problem raised by Rust folks in #3442.

**Reply by lfrancke** [reply]
Thank you, I'll read up on it.
